Update catalog-products-add-edit.md (#160)
PrestaShop/prestashop-specs: back-office/catalog/products/catalog-products-add-edit.md

* **Quantity**: 

**1.7.7.0**:
This field is not displayed for a product with combinations. This field is not displayed if stock management is disabled in Shop parameters &gt; Products settings.       
If simple product is selected, under the quantity field, there is a “quantities” link. When you click on it, you switch to quantities tab \(for a standard product\) or to virtual product tab \(for a virtual product\).       
When you update the field in basic settings tab, the same field in quantities \(for a standard product\) / virtual product tab is updated.       
The quantity is displayed in front-office if the option “Display available quantities on the product page” in Shop parameters &gt; Product settings is set to yes.          
This value is decremented when an order is placed except if the order status is Payement error. This value is incremented when the order status is cancelled, when you make a partial refund and check “Re-stock products” and when you return products and check “Re-stock products”.

**1.7.8.0**:

> The quantity is now only a text in semibold that displays the quantity. The quantity can not be updated through the basic setting tab.

Reviewer: Why not keeping the quantity field in basic settings tab ?

Author: Because we don't want the merchant to update the quantity on the product page. With the new wording on the quantity tab, we can bring awareness to the merchant.

> This field is not displayed for a product with combinations. This field is not displayed if stock management is disabled in Shop parameters &gt; Products settings.       

Reviewer: ```suggestion
The quantity is not displayed for a product with combinations. The quantity is not displayed if stock management is disabled in Shop parameters &gt; Products settings.       
```
We should change this part if it's not a field anymore

> If simple product is selected, under the quantity field, there is a “quantities” link. When you click on it, you switch to quantities tab \(for a standard product\) or to virtual product tab \(for a virtual product\).       

Reviewer 2: ```suggestion
If a simple product is selected, a “quantities” link should be displayed under the quantity field. Clicking on it will switch to quantities tab \(for a standard product\) or to virtual product tab \(for a virtual product\).       
```

> If simple product is selected, under the quantity field, there is a “quantities” link. When you click on it, you switch to quantities tab \(for a standard product\) or to virtual product tab \(for a virtual product\).       

Reviewer: ```suggestion
If simple product is selected, under the quantity, there is a “quantities” link. When you click on it, you switch to quantities tab \(for a standard product\) or to virtual product tab \(for a virtual product\).       
```
We should change this part if it's not a field anymore

The quantity is displayed in front-office if the option “Display available quantities on the product page” in Shop parameters &gt; Product settings is set to yes.          

> This value is decremented when an order is placed except if the order status is Payement error. This value is incremented when the order status is cancelled, when you make a partial refund and check “Re-stock products” and when you return products and check “Re-stock products”.

Reviewer 2: It is a bit unclear to me... there are three or two cases?

When you update the field in quantities tab, the same field in basic settings tab is updated.        
The quantity is displayed in front-office if the option “Display available quantities on the product page” in Shop parameters &gt; Product settings is set to yes.       
This value is decremented when an order is placed except if the order status is Payement error. This value is incremented when the order status is cancelled, when you make a partial refund and check “Re-stock products” and when you return products and check “Re-stock products”.

> On top of the quantity input, the page has the following wording (new wording integrated by the issue #19454): 

Reviewer: ```suggestion
On top of the quantity input, the page has the following wording (new wording integrated by the issue [#19454](https://github.com/PrestaShop/PrestaShop/issues/19454)): 
```

> "We advise you to modify the quantity directly in the [1]Catalog > Stocks[/1] section of the back office to avoid stock imbalance." in Admin.Notifications.Warning.

Reviewer: Maybe we should add this message also in the virtual product & combinations tabs ?

Author: You are right for the Pack and Virtual product. I'm going to see for the combinations tabs. Thanks

> The link redirects to the stock page on another tab with the product displayed/loaded.

Reviewer: Like if a search with the product name has been done to display only the good product ?

Author: Yes, exactly. So the merchant doesn't have to look for it

> The link redirects to the stock page on another tab with the product displayed/loaded.

Author: ```suggestion
The link redirects to the stock page on another tab with the product displayed as it was already searched.
```
